internal/db/p2p_schema_root.go

```diff
@@ -205,3 +176,52 @@ func (p *Peer) GetAllP2PCollections(ctx context.Context) ([]string, error) {
 
 	return collectionIDs, nil
 }
+
+func (db *db) PeerInfo() peer.AddrInfo {
+	db.peerMutex.RLock()
+	defer db.peerMutex.RUnlock()
+	if db.peerInfo.HasValue() {
+		return db.peerInfo.Value()
+	}
+	return peer.AddrInfo{}
+}
+
+func (db *db) loadP2PCollections(ctx context.Context) error {
```

Comment:
todo: this method's name is misleading. It's not just loading, but fires a bunch of events.

I think it should be renamed to something more appropriate.